<launch>
    <!-- Launch gazebo file which will start up gazebo and load controllers -->
    <include file="$(find hexapod_gazebo)/launch/gazebo.launch"/>

    <!-- Start up kinematic solver -->
    <!-- <node name="kinematics_solver" pkg="hexapod_control" type="kinematics_solver" respawn="false"/> -->

    <!-- Start up joint action servers -->
    <!-- <node name="leg_fr_joint_action_server" pkg="hexapod_control" type="leg_fr_joint_action_server" respawn="false"/>
    <node name="leg_fl_joint_action_server" pkg="hexapod_control" type="leg_fl_joint_action_server" respawn="false"/>
    <node name="leg_br_joint_action_server" pkg="hexapod_control" type="leg_br_joint_action_server" respawn="false"/>
    <node name="leg_bl_joint_action_server" pkg="hexapod_control" type="leg_bl_joint_action_server" respawn="false"/>
    <node name="tail_joint_action_server" pkg="hexapod_control" type="tail_joint_action_server" respawn="false"/> -->

    <!-- Start up pose action servers -->
    <!-- <node name="leg_fr_pose_action_server" pkg="hexapod_control" type="leg_fr_pose_action_server" respawn="true"/>
    <node name="leg_fl_pose_action_server" pkg="hexapod_control" type="leg_fl_pose_action_server" respawn="true"/>
    <node name="leg_br_pose_action_server" pkg="hexapod_control" type="leg_br_pose_action_server" respawn="true"/>
    <node name="leg_bl_pose_action_server" pkg="hexapod_control" type="leg_bl_pose_action_server" respawn="true"/>
    <node name="tail_pose_action_server" pkg="hexapod_control" type="tail_pose_action_server" respawn="true"/> -->

    <!-- Start up gait controllers -->
    <!-- <rosparam file="$(find hexapod_control)/config/gait_parameters.yaml" ns="/hexapod/gait"/>
    <node name="leg_fr_trajectory_action_server" pkg="hexapod_control" type="leg_fr_trajectory_action_server" respawn="false"/>
    <node name="leg_fl_trajectory_action_server" pkg="hexapod_control" type="leg_fl_trajectory_action_server" respawn="false"/>
    <node name="leg_br_trajectory_action_server" pkg="hexapod_control" type="leg_br_trajectory_action_server" respawn="false"/>
    <node name="leg_bl_trajectory_action_server" pkg="hexapod_control" type="leg_bl_trajectory_action_server" respawn="false"/>
    <node name="gait_controller" pkg="hexapod_control" type="gait_controller" respawn="false"/> -->

    <!-- Start up force sensor and stability stuff -->
    <!-- <node name="force_sensor_node" pkg="hexapod_control" type="force_sensor_node" respawn="true"/>
    <node name="stability_margin_node" pkg="hexapod_control" type="stability_margin_node" respawn="true"/>
    <node name="tail_controller" pkg="hexapod_control" type="tail_controller" respawn="true"/> -->
</launch>